mcs/class/corlib/Test/System/TimeZoneInfoTest.cs

```diff
@@ -467,6 +478,25 @@ public void ConvertTimeToUtc_Overflow ()
 			}
 
 			[Test]
+			public void ConvertFromToUtc_LocalAsUtc ()
+			{
+				var oldLocal = TimeZoneInfo.Local;
+				TimeZoneInfoTest.SetLocal (TimeZoneInfo.Utc);
+
+				try {
+					DateTime utc = DateTime.UtcNow;
+					Assert.AreEqual(utc.Kind, DateTimeKind.Utc);
+					DateTime converted = TimeZoneInfo.ConvertTimeFromUtc(utc, TimeZoneInfo.Local);
+					Assert.AreEqual(DateTimeKind.Local, converted.Kind);
+					DateTime back = TimeZoneInfo.ConvertTimeToUtc(converted, TimeZoneInfo.Local);
+					Assert.AreEqual(back.Kind, DateTimeKind.Utc);
+					Assert.AreEqual(utc, back);
```

Comment:
code guidelines: space before opening parenthesis

nit: while you're at it, the commit message of this commit has a typo: `#334711` instead of `#33471`.